couchbase.py

```diff
@@ -303,6 +304,20 @@ def _parse_metrics(obj_to_parse, dimensions, request_type, module_config):
                     if metric:
                         metrics.append(metric)
 
+        if 'hot_keys' in obj_to_parse:
+            hot_keys = obj_to_parse['hot_keys']
+            key = 0
+            metric_name_pref = 'bucket.hot_keys'
+            for keys in hot_keys:
+                for name, value in keys.items():
+                    if isinstance(value, numbers.Number):
+                        key += 1
+                        metric = _process_metric(metric_name_pref, str(key),
```

Comment:
The key name could change very often depending of Couchbase usage. In a couple of seconds a key never called could become a hot_key and Conversely. I think if we return the name of the key, we will have a very huge list.